test/framework/k8s_cluster.go

```diff
@@ -313,6 +314,7 @@ func (c *K8sCluster) yamlForKumaViaKubectl(mode string) (string, error) {
 
 	if c.opts.zoneEgress {
 		argsMap["--egress-enabled"] = ""
+		args = append(args, "--set", fmt.Sprintf("%segress.logLevel=debug", Config.HelmSubChartPrefix))
```

Comment:
we already do this on uni https://github.com/kumahq/kuma/blob/3341716b748569b4ab2f6ead2824e38cf57e4921/test/framework/universal_app.go#L481